### PR TITLE
fix: formatting issues on blog

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-9090

image

Compare: 
http://localhost:3000/learn/jumper-loyalty-pass/
http://jumper.exchange/learn/jumper-loyalty-pass/

### CODE REVIEW

File: src/components/Blog/CustomRichBlocks.tsx [hunk 1]
```diff
@@ -195,18 +195,41 @@ export const CustomRichBlocks = ({
...
-                      italic={el.props.italic}
-                      strikethrough={el.props.strikethrough}
-                      underline={el.props.underline}
-                      bold={el.props.bold}
-                      key={`blog-paragraph-${index}`}
-                    >
-                      {el.props.text}
-                    </BlogParagraph>
-                  );
+                  if (el.props.text.indexOf('\n') === -1) {
```
Comment (oktapodia) [inline src/components/Blog/CustomRichBlocks.tsx:198]: Not really fan of this, what is the purpose of it exactly? you can use `nl2br` to do this

**Review by oktapodia — CHANGES_REQUESTED** [review]

File: src/components/Blog/CustomRichBlocks.tsx (hunk 1 above)
Comment (dennyscode) [reply src/components/Blog/CustomRichBlocks.tsx:202]: Thanks, Sebastien! This condition should be met if there is no line-break within the strapi-paragraph.
In such cases we just return our default <BlogParagraph>. Compared to, if there is a "\n" found, it is split into an array and returns multiple <BlogParagraph>-components.

Feel free to share some more info on how to apply nl2br.

Comment (oktapodia) [reply src/components/Blog/CustomRichBlocks.tsx:202]: something like this https://github.com/yosuke-furukawa/react-nl2br

Comment (dennyscode) [reply src/components/Blog/CustomRichBlocks.tsx]: Thanks, good idea! Added it now

**Review by admsk2 — APPROVED** [review]

File: src/components/Blog/CustomRichBlocks.tsx [hunk 2]
```diff
@@ -199,18 +200,32 @@ export const CustomRichBlocks = ({
...
-                      bold={el.props.bold}
-                      key={`blog-paragraph-${index}`}
-                    >
-                      {el.props.text}
-                    </BlogParagraph>
+                  const nl2brText: Array<ReactElement | string> = nl2br(
+                    el.props.text,
                   );
+                  return nl2brText.map((line, lineIndex: number) => {
+                    if (line === '') {
```
Comment (oktapodia) [inline src/components/Blog/CustomRichBlocks.tsx:207]: why is that needed?

**Review by oktapodia — COMMENTED** [review]
quick question

File: src/components/Blog/CustomRichBlocks.tsx (hunk 2 above)
Comment (dennyscode) [reply src/components/Blog/CustomRichBlocks.tsx]: Thanks! I removed that part

**Review by oktapodia — APPROVED** [review]
LGTM
